src/components/menu-top/MenuTop.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <div class="menu-top">
     <b-navbar type="is-primary">
+      <template slot="brand">
+        <b-navbar-item>
+          <img
+            src="/images/highstepper-gold.png"
```

Comment:
Sorry there’s no example for images, but can you take a look at using relative file paths instead? https://cli.vuejs.org/guide/html-and-static-assets.html#url-transform-rules
This way we won’t have conflicts when we deploy this to github pages (the file will be located at /calchart-redesign/images/highstepper-gold.png)
```suggestion
            src="@/images/highstepper-gold.png"
```

Reply:
Changed!